cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -583,7 +583,15 @@
   public static final class Notification {
     public static final String TRANSPORT_SYSTEM = "notification.transport.system";
 
-    public static final String KAFKA_DELEGATE_NOTIFICATION_SERVICE = "kafka.delegate.notification.service";
+    /**
+     * Notifications in Streams constants.
+     */
+    public static final class Stream {
+      public static final String STREAM_HEARTBEAT_FEED_CATEGORY = "streamHeartbeats";
```

Comment:
We had that discussion with Terence and Andreas when we were originally designing the Notification system. And, even though I don't remember the specifics, it seemed better to keep the category of a feed as a string rather than an enum. I think it was if, at some point we wanted to expose the feeds to the user, then they could easily define feeds with their own categories.